Better structure of extras documentation (#50495)
mistercrunch/Airflow: airflow-core/docs/extra-packages-ref.rst

These are core Airflow extras that extend capabilities of core Airflow. They usually do not install provider

> packages (with the exception of ``celery`` and ``cncf.kubernetes`` extras), they just install necessary

Reviewer: ```suggestion
These are core Airflow extras that extend capabilities of core Airflow. They do not install provider
packages, they just install necessary
```

We don't have celery of k8s in the list, so we can probably simplify this a bit?

Author: Ah.. left-over :)

+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| sentry              | ``pip install 'apache-airflow[sentry]'``            | Sentry service for application logging and monitoring                      |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| standard            | ``pip install apache-airflow[standard]'``           | Standard hooks and operators                                               |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| statsd              | ``pip install 'apache-airflow[statsd]'``            | Needed by StatsD metrics                                                   |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+

Meta-airflow package extras
---------------------------

> Airflow 3 is released in several packages. the ``apache-airflow`` package is a meta-package that installs

Reviewer: ```suggestion
Airflow 3 is released in several packages. The ``apache-airflow`` package is a meta-package that installs
```

> all the other packages when you run airflow as a standalone installation, and it also has several extras
> that are not extending airflow core functionality, but they are useful for the users who want to install

Reviewer: ```suggestion
all the other packages when you run Airflow as a standalone installation, and it also has several extras
that are not extending Airflow core functionality, but they are useful for the users who want to install
```

+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| pandas              | ``pip install 'apache-airflow[pandas]'``            | Install Pandas library compatible with Airflow                             |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| polars              | ``pip install 'apache-airflow[polars]'``            | Polars hooks and operators                                                 |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| rabbitmq            | ``pip install 'apache-airflow[rabbitmq]'``          | RabbitMQ support as a Celery backend                                       |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+

> | saml                | ``pip install 'apache-airflow[saml]'``              | Support for SAML authentication in Amazon provider                         |
> +---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
> | s3fs                | ``pip install 'apache-airflow[s3fs]'``              | Support for S3 as Airflow FS                                               |

Reviewer: ```suggestion
| s3fs                | ``pip install 'apache-airflow[s3fs]'``              | Support for S3 as Airflow FS                                               |
+---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
| saml                | ``pip install 'apache-airflow[saml]'``              | Support for SAML authentication in Amazon provider                         |
```

Let's do alphabetical.

Author: Depending which alphabet you choose :)
